### PR TITLE
Nav bar

### PR DESCRIPTION
**Pull request by BlandineM**

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,8 +1,13 @@
 import React from "react";
 import "./components/reset.css";
+import Nav_bar from "./components/Nav_bar";
```
Comment (LeoDiduch) [inline frontend/src/App.js]: change file call

File: frontend/src/App.js [hunk 2]
```diff
@@ -1,8 +1,13 @@
...
 
 function App() {
-  return <div className="App"></div>;
+  return (
+    <div className="App">
+      <Nav_bar />
```
Comment (LeoDiduch) [inline frontend/src/App.js]: change file call

**Review by LeoDiduch — CHANGES_REQUESTED** [review]
##Blocking :

Please change files name "Nav_bar" is not a good practice for me, instead call his "NavBar" file with a capital letter with each new word.
And don't forget to change the calls in the others files.

**Review by LeoDiduch — DISMISSED** [review]

**Review by LeoDiduch — DISMISSED** [review]

File: frontend/src/components/NavBar.jsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import "./NavBar.scss";
+
+function Nav_bar() {
+  return (
+    <header>
+      <div className="navBar">
```
Comment (LoicBrassart) [inline frontend/src/components/NavBar.jsx:7]: Hmm. That's just a suggestion, but I think you could have given the `navBar` class to your header, or used a `<nav>` tag to mark your navigation menu

File: frontend/src/components/NavBar.scss [hunk 4]
```diff
@@ -0,0 +1,17 @@
+header {
+  display: flex;
+  justify-content: center;
+
+  .navBar {
+    display: flex;
+    justify-content: space-around;
+    width: 75vw;
+    img {
+      transform: scale(0.95);
```
Comment (LoicBrassart) [inline frontend/src/components/NavBar.scss:10]: Still another recommendation, but for consistency's sake, you should use similar values for all these transforms (I've seen that somewhere, haven't I ? :p )
Here you use a 0.95 factor but in another file you use a 0.90 one IIRC

**Review by LoicBrassart — DISMISSED** [review]

**Review by LeoDiduch — DISMISSED** [review]

**Review by LoicBrassart — DISMISSED** [review]
Ok!

**Review by Adwarf14 — APPROVED** [review]

**Pull request merged**